docker/4.4.0-alpine/healthcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -x -e -u
+
+# Simple healtcheck on BK port
+nc -z localhost ${BK_PORT}
```

Comment:
Interesting suggestion. Doing it every 3 seconds doesn't impact performances, right?